Marlin/Marlin_main.cpp

```diff
@@ -868,7 +872,28 @@ void setup() {
  *  - Call LCD update
  */
 void loop() {
-  if (commands_in_queue < BUFSIZE) get_available_commands();
+
+
+if (filamentRemove) {
```

Comment:
I'm just using EEPROM because the variable was used in the main loop.
And i have it there because i thugth that there were the right way but i could easily move to ultralcd.cpp file. What you think?